debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/MySqlAntlrDdlParser.java

```diff
@@ -215,7 +215,8 @@ public String parseName(MySqlParser.UidContext uidContext) {
     public TableId parseQualifiedTableId(MySqlParser.FullIdContext fullIdContext) {
         String fullTableName = fullIdContext.getText();
         int dotIndex;
-        if ((dotIndex = fullTableName.indexOf(".")) > 0) {
+        // The quoted table name that contains dot means that it is just a part of the name of the table
+        if (!isQuoted(fullTableName) && ((dotIndex = fullTableName.indexOf(".")) > 0)) {
```

Comment:
Couldn't we encounter something like ``inventory.`My.Table` ``?

Reply:
@gunnarmorling Just fixed